jenkins/ci.suse.de/templates/cloud-socok8s-ansible-lint.yaml

```diff
@@ -0,0 +1,20 @@
+- job-template:
```

Comment:
(nit) I would have just put this in the same file as the other socok8s tests (although that should probably be renamed from integrations)

Reply:
can we? I dont know how :D

Reply:
Just another entry in the file. I was about to point you to another file to look at, but it looks like they are all separate anyway, so this follows convention :-).